content/_snippets/tx-metadata-field-table.md

```diff
@@ -0,0 +1,7 @@
+| Field                                   | Value               | Description  |
+|:----------------------------------------|:--------------------|:-------------|
+| `AffectedNodes`                         | Array               | List of [ledger objects](ledger-object-types.html) that were created, deleted, or modified by this transaction, and specific changes to each. |
+| `DeliveredAmount`                       | [Currency Amount][] | _(May be omitted)_ For a [partial payment](partial-payments.html), this field records the amount of currency actually delivered to the destination. For more convenient processing, instead use the `delivered_amount` field, which is provided for all Payment transactions, partial or not. |
```

Comment:
> For more convenient processing, instead use the `delivered_amount` field, which is provided for all Payment transactions, partial or not.

Can this be emphasized more somehow? I know it's covered elsewhere but the wording here makes it seem more like a matter of convenience than something potentially more consequential.

Reply:
Changed to:

> To avoid errors when reading transactions, instead use...